### PR TITLE
Update CodeMirror Spell Checker file locations

### PR DESCRIPTION
**Pull request by WesCossick**
Moved to `/dist/` folder

### CODE REVIEW

**Comment by WesCossick** [comment]
Looks like there's an error, it's valid JSON.

File: files/codemirror.spell-checker/update.json [hunk 1]
```diff
@@ -3,6 +3,6 @@
   "name": "codemirror-spell-checker",
   "repo": "NextStepWebs/codemirror-spell-checker",
   "files": {
-    "include": ["spell-checker.min.js", "spell-checker.min.css", "en_US.aff", "en_US.dic"]
+    "include": ["dist/spell-checker.min.js", "dist/spell-checker.min.css", "dist/en_US.aff", "dist/en_US.dic"]
```
Comment (megawac) [inline files/codemirror.spell-checker/update.json]: Could you change this to just setting the `basePath` to `dist`?

**Comment by WesCossick** [comment]
I updated the JSON to use `basePath` for both my packages.

**Comment by megawac** [comment]
@jsdelivrbot squish & squash

File: files/codemirror.spell-checker/update.json [hunk 2]
```diff
@@ -3,6 +3,7 @@
   "name": "codemirror-spell-checker",
   "repo": "NextStepWebs/codemirror-spell-checker",
   "files": {
+    "basePath": "dist",
     "include": ["spell-checker.min.js", "spell-checker.min.css", "en_US.aff", "en_US.dic"]
```
Comment (jimaek) [inline files/codemirror.spell-checker/update.json]: There are no such files in the repo https://github.com/jsdelivr/jsdelivr/pull/6460

**Comment by WesCossick** [comment]
Fixed. Typo on filenames.

**Comment by jimaek** [comment]
I still see the old files. You need to either replace the files in 1.0.5 or release 1.0.6

**Comment by WesCossick** [comment]
I replaced the files in the release 1.0.5.

**Pull request closed without merging**